src/lv_objx/lv_ta.c

```diff
@@ -955,17 +955,15 @@ const lv_style_t * lv_ta_get_style(const lv_obj_t * ta, lv_ta_style_t type)
 void lv_ta_get_selection(lv_obj_t * ta, int * sel_start, int * sel_end)
```

Comment:
This function might not be required once we have functions to get "start" and "end". IMO they are more convenient to use.